app/services/users.js

```diff
@@ -0,0 +1,35 @@
+const bcrypt = require('bcryptjs');
+const logger = require('../logger');
+const { databaseError } = require('../errors');
+const { user: userModel } = require('../models');
+
+const create = async user => {
+  try {
+    const hash = await bcrypt.hash(user.password, 12);
+    const createdUser = (await userModel.create({ ...user, password: hash })).toJSON();
+    delete createdUser.password;
+    return createdUser;
+  } catch (error) {
+    logger.error(error);
+    throw databaseError('Cannot create user');
+  }
+};
+
+const findByEmail = async email => {
+  try {
+    const user = await userModel.findOne({
+      attributes: { exclude: ['password'] },
```

Comment:
I think you dont use this line, you only need the email, and the result of this function you never send as request response